### DISCUSSION TITLE
Some more Policies

### DISCUSSION THREAD

**Discussion by WildByDesign** [TheBobPony/MSEdgeTweaker#2] [Ideas]
@TheBobPony  This is a great idea. Thank you for sharing this script. I wanted to share some of my Edge policies with you for possible inclusion because Edge would not be usable for me without many of these.

I don't have time to go through my policies right now to remove some of the less relevant ones. Particularly, the ones to do with sidebar, hub, searchbar, etc. are pure gold. The Edge UX/UI is significantly more responsive, snappy and less visual bloat.

The local-ntp is great because it also loads way faster in comparison. Unfortunately, MS just crappified the local-ntp in version 126. So I just made a replica of the original local-ntp as a temporary measure. I could share that if there is any interest. Changing `NewTabPageLocation` (and a few other policies) require a Domain-joined device or faking it (https://hitco.at/blog/apply-edge-policies-for-non-domain-joined-devices/).

I have just copied the registry export as-is at the moment:

```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge]
"ShowHomeButton"=dword:00000001
"BackgroundModeEnabled"=dword:00000000
"AutofillCreditCardEnabled"=dword:00000000
"BrowserGuestModeEnabled"=dword:00000000
"PasswordManagerEnabled"=dword:00000000
"AutofillAddressEnabled"=dword:00000000
"UserFeedbackAllowed"=dword:00000000
"FavoritesBarEnabled"=dword:00000001
"BrowserAddProfileEnabled"=dword:00000000
"DnsOverHttpsMode"="off"
"TrackingPrevention"=dword:00000000
"WebCaptureEnabled"=dword:00000000
"EdgeCollectionsEnabled"=dword:00000000
"EdgeShoppingAssistantEnabled"=dword:00000000
"StartupBoostEnabled"=dword:00000000
"BuiltInDnsClientEnabled"=dword:00000000
"NewTabPageLocation"="chrome-search:local-ntp/local-ntp.html"
"LocalBrowserDataShareEnabled"=dword:00000000
"HubsSidebarEnabled"=dword:00000000
"NewPDFReaderEnabled"=dword:00000001
"ShowAcrobatSubscriptionButton"=dword:00000000
"SearchbarAllowed"=dword:00000000
"SearchbarIsEnabledOnStartup"=dword:00000000
"ExperimentationAndConfigurationServiceControl"=dword:00000000
"RendererAppContainerEnabled"=dword:00000001
"NetworkServiceSandboxEnabled"=dword:00000001
"SearchInSidebarEnabled"=dword:00000002
"StandaloneHubsSidebarEnabled"=dword:00000000

```

Thanks to your easy-to-use script, more users will enjoy a better, faster Edge. Cheers!

**Comment by TheBobPony** [comment]
Thank you very much for your feedback. Will look into adding your suggestions into the script in a later update!
